src/Provider/UnitTests/Provider.Web/Extensions/ServiceClaimExtensionsTests.cs

```diff
@@ -0,0 +1,145 @@
+using FluentAssertions;
+using Xunit;
+
+namespace Esfa.Recruit.Provider.UnitTests.Provider.Web.Extensions
+{
+    public class ServiceClaimExtensionsTests
+    {
+        [Theory]
+        [InlineData("DAA", true)]
+        [InlineData("DAB", true)]
+        [InlineData("DAC", true)]
+        [InlineData("DAV", true)]
+        [InlineData("DA", false)]
+        [InlineData("FAKE", false)]
+        public void ValidateServiceClaimsTest(string claim, bool expected)
+        {
+            bool result = claim.IsServiceClaim();
+            result.Should().Be(expected);
+        }
+
+        [Theory]
+        [InlineData(ServiceClaim.DAA, true)]
+        [InlineData(ServiceClaim.DAB, true)]
+        [InlineData(ServiceClaim.DAC, true)]
+        [InlineData(ServiceClaim.DAV, true)]
+        public void ShouldReturnTheHighestClaim_DAA(ServiceClaim minimumServiceClaim, bool expected)
+        {
+            var claims = new List<Claim>
+            {
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAV"),
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAA"),
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAB"),
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAC")
+            };
+
+            var identity = new ClaimsIdentity(claims, "TestAuthType");
+            var claimsPrincipal = new ClaimsPrincipal(identity);
+
+            var result = claimsPrincipal.HasPermission(minimumServiceClaim);
+            result.Should().Be(expected);
+        }
+
+        [Theory]
+        [InlineData(ServiceClaim.DAA, true)]
+        [InlineData(ServiceClaim.DAB, true)]
+        [InlineData(ServiceClaim.DAC, true)]
+        [InlineData(ServiceClaim.DAV, true)]
+        public void ShouldReturnHasPermission_DAA(ServiceClaim minimumServiceClaim, bool expected)
+        {
+            var claims = new List<Claim>
+            {
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAV"),
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAA"),
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAB"),
+                new Claim(ProviderRecruitClaims.IdamsUserServiceTypeClaimTypeIdentifier, "DAC")
+            };
+
+            var identity = new ClaimsIdentity(claims, "TestAuthType");
+            var claimsPrincipal = new ClaimsPrincipal(identity);
+
+            var result = claimsPrincipal.HasPermission(minimumServiceClaim);
+            result.Should().Be(expected);
+        }
+
+        [Theory]
+        [InlineData(ServiceClaim.DAA, false)]
+        [InlineData(ServiceClaim.DAB, true)]
+        [InlineData(ServiceClaim.DAC, true)]
+        [InlineData(ServiceClaim.DAV, true)]
+        public void ShouldReturnHasPermission_DAB(ServiceClaim minimumServiceClaim, bool expected)
```

Comment:
Could these all be done in 1 test with another param in the inline data? minimumServiceClaim, actualServiceClaim, expected. Or something along these lines? Removes the need to know which claims to add/remove from the List<Claim>